cmd/meroxa/root/pipelines/create.go

```diff
@@ -48,29 +44,15 @@ type Create struct {
 	args struct {
 		Name string
 	}
-
-	flags struct {
```

Comment:
How come we're removing this?

Reply:
When discussing with Diana, she mentioned it was either unused in the pipeline code or had no use case for being exposed to customers.